integration-cli/docker_utils.go

```diff
@@ -338,11 +339,14 @@ func sockRequest(method, endpoint string, data interface{}) (int, []byte, error)
 
 	res, body, err := sockRequestRaw(method, endpoint, jsonData, "application/json")
 	if err != nil {
-		b, _ := ioutil.ReadAll(body)
+		if body != nil {
+			b, _ = ioutil.ReadAll(body)
+		}
 		return -1, b, err
 	}
-	var b []byte
-	b, err = readBody(body)
+	if body != nil {
```

Comment:
That's fair enough. I found the panic on the first check, just blindly updated both. Will revert the second one.